packages/core/src/VDocumentMap.ts

```diff
@@ -1,41 +1,42 @@
 import { Format } from '../../utils/src/Format';
 import { VNode } from './VNodes/VNode';
 
-const fromDom = new Map<Node, VNode[]>();
-const toDom = new Map<VNode, [Node, number]>();
-
-export const VDocumentMap = {
+export class VDocumentMap {
+    _fromDom = new Map<Node, VNode[]>();
+    _toDom = new Map<VNode, [Node, number]>();
```

Comment:
I would have completely replaced the `VDocumentMap` by a map `Map<Node, VNode[]>` used only by the parser and a map `Map<VNode, [Node, number]>` use only by the renderer.